### ISSUE TITLE
Invitation email was not sent. Please try resending.

### ISSUE THREAD

**Issue by Leonerdy** [felixrieseberg/Ghost-Azure#72]
I'm trying invite friends to blog but I have this error "Invitation email was not sent. Please try resending.", I configure in azure like in tutorial https://docs.ghost.org/docs mail config, emailService: Mailgun, emailUserName: postmaster@sandbox... and emailPassword: ****, I  can send email test in aba Labs for my email but I couldnt resolve this problem, some help please.

**Comment by DevOpsBoondoggles** [comment]
Heya, 
not sure specifically but mine works fine with Sendgrid, I added a wikipage with instructions if that's any good

https://github.com/felixrieseberg/Ghost-Azure/wiki/Setup-Free-Mail-sender-for-the-blog-with-Sendgrid

**Comment by Leonerdy** [comment]
yes, I solve my problem with sendgrid, i forgot put the solution here, tnks for your comment.

**Comment by chris-brown** [comment]
Do send grid have free plans? I believe mailgun.com do?

**Comment by DevOpsBoondoggles** [comment]
Something like the first million a month are free if you sign up through azure so you’ll be okay I wager 

Thanks
Gabriel
